xml/admin_upgrade.xml

```diff
@@ -259,4 +259,218 @@ value: value)}'</command></screen>
    </procedure>
   </sect2>
  </sect1>
+ 
+ <sect1 xml:id="sec.deploy.upgrade.caasp2">
+  <title>Upgrading from &productname; 2</title>
+  <note>
+   <para>
+    If you wish to upgrade to &productname; v3 from v1 you must first perform
```

Comment:
The title says CaaSP v2, but here we say from v1. As we don't support v1 -> v3, I guess the v1 should be v2.